pkg/plugins/runtime/k8s/controllers/configmap_controller.go

```diff
@@ -131,6 +137,22 @@ func (m *DataplaneToMeshMapper) Map(obj kube_handler.MapObject) []kube_reconile.
 		return nil
 	}
 
+	dp := core_mesh.NewDataplaneResource()
+	if err := m.ResourceConverter.ToCoreResource(cause, dp); err != nil {
+		converterLog.Error(err, "failed to parse Dataplane", "dataplane", cause.Spec)
+		return nil
+	}
+
+	if dp.Spec.IsIngress() {
+		var requests []kube_reconile.Request
+		for _, service := range dp.Spec.GetNetworking().GetIngress().GetAvailableServices() {
+			requests = append(requests, kube_reconile.Request{
+				NamespacedName: kube_types.NamespacedName{Namespace: m.SystemNamespace, Name: vips.ConfigKey(service.Mesh)},
+			})
```

Comment:
Will the subsequent requests be dropped? we could build a map of meshes

Reply:
I'm not sure, but this is a good point, let's reduce work for the controller and put meshes into the set